app/scripts/components/data-catalog/container.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { getString } from 'veda';
 import { allDatasets } from '$components/exploration/data-utils';
```

Comment:
Nit: To avoid coupling features, should we create a `common/data-utils.ts` to move the `allDatasets` util there and use it in both, the exploration and the data-catalog?